components/survey/Card/index.jsx

```diff
@@ -0,0 +1,47 @@
+import Link from 'next/link';
+import PropTypes from 'prop-types';
+import Image from 'next/image';
+
+import forwardButton from '@/public/angle-right.svg';
+
+export const surveyCardTestIds = {
+  base: 'survey-card',
+  coverImageUrl: 'survey-card__cover_image_url',
+  title: 'survey-card__title',
+  description: 'survey-card__description',
+  forwardButton: 'survey-card__forward',
+};
+
+const SurveyCard = ({ slideId, title, description, coverImageUrl }) => (
+  <div data-test-id={surveyCardTestIds.base}>
+    <div className="w-[704px] h-[302px] mb-8" data-test-id={surveyCardTestIds.coverImageUrl}>
+      <Image className="rounded-xl" src={coverImageUrl} alt="cover image" width={704} height={302} />
+    </div>
+    <div className="flex justify-between mb-[155px]">
+      <div>
+        <div className="font-extrabold text-white text-base-xxl mb-2" data-test-id={surveyCardTestIds.title}>
+          {title}
+        </div>
+        <div className="text-white/60 text-base-large" data-test-id={surveyCardTestIds.description}>
+          {description}
+        </div>
+      </div>
+      <div className="w-14 h-14" data-test-id={surveyCardTestIds.forwardButton}>
+        <Link href={`/surveys/${slideId}`}>
+          <a>
+            <Image className="cursor-pointer" src={forwardButton} alt="forward" />
```

Comment:
```suggestion
            <Image className="cursor-pointer" src={forwardButton} alt="next survey" />
```

I like the exercise of imagining my web browser cannot display images and instead just display the `alt` message, then I think about what would be the most explicit word/expression in the context of the page.
Sure "forward" is not "wrong" (the icon is a forward icon), but it does not feel natural in the context of a survey carousel. 

Doing this exercise can bring a lot of value for users in need of a screen reader (e.g. blind people) as well as the bots parsing your app/website.

Reply:
Thanks for the detailed reply, done in 86c458775e4bb47df1b72df50f6dd54aba632aa2!